R/draft_pick.R

```diff
@@ -19,13 +22,22 @@
 library(tidyverse)
 library(httr)
 library(jsonlite)
+library(testthat)
```

Comment:
I think we are supposed to add our functions as imports to the roxygen documentation by doing:

```
#' @import tidyverse
#' @import httr
#' @import jsonlite
#' @import testthat
```

```suggestion
```

Reply:
then in the directory with the function run this to add the library to the Namespace:

`devtools::document()`

Reply:
Oh, ok. I didn't know that! Thank you!